### ISSUE TITLE
drsolver --gpu selection does not work (V0.92)

### ISSUE THREAD

**Issue by ghmgit** [drnick23/drplotter#40]
Hi,

the --gpu option does not work. Despite I specify "0" it still asks:

```
root@3abf31929cac:/# drsolver --gpu 0
Using client token from DRPLOTTER_CLIENT_TOKEN environment variable: xxx
GPU Selection:
--------------------------------
     0 | NVIDIA GeForce RTX 4090    |    23812 / 24217 MiB |  AVAILABLE
     1 | Quadro RTX 4000            |      7855 / 7966 MiB |    LOW RAM
Select a GPU (0-1): ^C

**Comment by ghmgit** [comment]
Until this is fixed I am using this workaround:
```
root@3abf31929cac:/# export CUDA_VISIBLE_DEVICES=0

root@3abf31929cac:/# drsolver       
Using client token from DRPLOTTER_CLIENT_TOKEN environment variable: xxx
... .NVIDIA GeForce RTX 4090) selected.

**Comment by drnick23** [comment]
This is fixed in the 0.10.2 release.
